docs/layouts/partials/navbar.html

```
@@ -0,0 +1,75 @@
+  <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+    {{- /**/ -}}
+    <span class="navbar-brand__logo navbar-logo">
+      {{- if ne .Site.Params.ui.navbar_logo false -}}
+        {{ with resources.Get "icons/logo.svg" -}}
+          {{ ( . | minify).Content | safeHTML -}}
+        {{ end -}}
+      {{ end -}}
+    </span>
+    {{- /**/ -}}
+    <span class="navbar-brand__name" aria-label="{{ .Site.Title }}">
+      <img src="{{ "images/title.svg" | relURL }}" alt="{{ .Site.Title }}" style="height:2.1rem;width:auto;">
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)